### DISCUSSION TITLE
Some thinking about QDrawer in mini mode

### DISCUSSION THREAD

**Discussion by 549393092** [quasarframework/quasar#14855] [General - Components / Directives / etc]
Dear Quasar team,

First of all, yes, I like QDrawer in mini mode, it makes the main screen bigger and done it automatically.
But when using QDrawer, I found it's inconvenient if user do some operation near the border of QDrawer.
Take the demo for example,
https://v1.quasar.dev/layout/drawer#example--mini-mode-with-mouseover-mouseout-trigger
If user want to copy the first words "Lorem ipsum dolor" and he moves the mouse from right to left, then it's easy to touch the area of QDrawer and make it expanded.

So is that possible to make a delay, for example one second delay, after that time, if the cursor is still in QDrawer, then to trigger it expand. Or do you have better proposal, thanks!

Regards

**Comment by MilosPaunovic** [comment]
Set a delay for your event handlers for mini mode:

```js
@mouseover="miniState = false"
@mouseout="miniState = true"
```
